app/domain/queries/find_content.rb

```diff
@@ -60,7 +60,11 @@ def array_to_hash(array)
 
   def slice_editions
     editions = Dimensions::Edition.relevant_content
-    editions = editions.where('organisation_id = ?', organisation_id)
+    if organisation_id == 'none'
```

Comment:
What about adding the filter by organisation as a `scope` for editions? It feels like something to be reused in other places?

Reply:
We already had one - I just didn't use it.